src/main/java/org/jpeek/Skeleton.java

```diff
@@ -344,6 +351,23 @@ public void visitFieldInsn(final int opcode,
                     }
                     Skeleton.Visitor.this.dirs.set(attr).up().up().up().up();
                 }
+
+                @Override
+                public void visitMethodInsn(final int opcode,
```

Comment:
@llorllale what does ``insn`` mean? Can you leave a JavaDoc over this method?

Reply:
`insn` stands for "instruction" - we're overriding [MethodVisitor.visitMethodInsn](http://asm.ow2.org/asm60/javadoc/user/org/objectweb/asm/MethodVisitor.html#visitMethodInsn-int-java.lang.String-java.lang.String-java.lang.String-boolean-).

Given `@Override` and this explanation, do you still feel a javadoc is needed?